### PR TITLE
refactor(ios): remove old timer hack for Firefox iOS6, remove old tests

### PR DESCRIPTION
**Pull request by vladikoff**
Fixes https://github.com/mozilla/fxa-content-server/issues/5820

@vbudhram r?

### CODE REVIEW

**Comment by vladikoff** [comment]
@vbudhram r? 

Tested with local Firefox iOS and seems to work

File: tests/functional/fx_ios_v1_sign_in.js [hunk 1]
```diff
@@ -100,43 +100,6 @@ registerSuite('FxiOS v1 signin', {
...
-        // email 2 - "You have verified your Firefox Account"
-        .then(openVerificationLinkInNewTab(email, 1, {query}))
-        .then(switchToWindow(1))
-        .then(testElementExists(selectors.CONNECT_ANOTHER_DEVICE.HEADER))
-        .then(closeCurrentWindow())
-
-        .then(testElementExists(selectors.CONFIRM_SIGNUP.HEADER));
-    },
-
-    'Fx iOS >= 6.1 unverified, verify same browser': function () {
```
Comment (vbudhram) [inline tests/functional/fx_ios_v1_sign_in.js]: Should we keep this test? There doesn't appear to be coverage for `unverified, verify same browser`.

File: tests/functional/fx_ios_v1_sign_up.js [hunk 2]
```diff
@@ -70,29 +46,6 @@ registerSuite('FxiOS v1 sign_up', {
...
-        .then(setupTest(UA_STRINGS['ios_firefox_6_0']))
-
-        // no screen transition, no polling even, in Fx <= 6.0
-        .then(noPageTransition(selectors.CONFIRM_SIGNUP.HEADER))
-
-        // A post-verification email should be sent, this is Sync.
-        .then(testEmailExpected(email, 1));
-    },
-
-    'Fx iOS >= 6.1 sign up, verify same browser': function () {
```
Comment (vbudhram) [inline tests/functional/fx_ios_v1_sign_up.js]: Same as above, should we keep as `sign up, verify same browser`?

File: tests/functional/fx_ios_v1_sign_in.js (hunk 1 above)
Comment (vladikoff) [reply tests/functional/fx_ios_v1_sign_in.js]: Yeap!

File: tests/functional/fx_ios_v1_sign_up.js (hunk 2 above)
Comment (vladikoff) [reply tests/functional/fx_ios_v1_sign_up.js]: we have `Fx iOS >= 11.0 sign up, verify same browser` below which does the same but with CWTS so we can probably avoid repeating this one

**Pull request merged**